### PR TITLE
Time issue/ code overide

### PR DESCRIPTION
**Pull request by Akhill2020**
**Description:** Code is is re-updated for time issue on the calendar 
**Trello:** https://trello.com/c/t3qrDxX8/4-2-seems-like-our-release-reverted-this-change

### CODE REVIEW

**Comment by rosinghal** [comment]
@Akhill2020 lint failing, all good otherwise.

File: readme.txt [hunk 1]
```diff
@@ -97,6 +97,11 @@ We'd love your help! Here's a few things you can do:
 
 == Changelog ==
 
+= 3.2.2 =
+*  Dev: Update google api client library to 2.x.
```
Comment (rosinghal) [inline readme.txt:101]:
```suggestion
*  Dev: Update Google API client library to 2.13.1.
```

File: includes/admin/pages.php [hunk 2]
```diff
@@ -272,7 +272,10 @@ public function html()
     		}
     	}
     	// It will display when the Pro is activated.
-    	if (!is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php')) { ?>
+    	if (
+    		!is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php') &&
```
Comment (rosinghal) [inline includes/admin/pages.php:276]: We don't need this condition. Only the following state should be enough. Does this mean everybody who got the release earlier was not getting the correct message?

Comment (rosinghal) [reply includes/admin/pages.php:276]: Can we check for class name instead of folder?

Comment (rosinghal) [reply includes/admin/pages.php:276]: I see folder as `simple-calendar-google-calendar-pro` in our demo.

Comment (Akhill2020) [reply includes/admin/pages.php:276]: I have tried with class name but it is not working may be due to SC is loaded before the pro addon. And this is actually a right way to check plugins exist in WordPress. 
I am not sure why he use the "main" in plugin folder maybe he got the folder name like this when he clone from git so I have update the condition.

File: readme.txt (hunk 1 above)
Comment (Akhill2020) [reply readme.txt:101]: Done

File: includes/admin/pages.php (hunk 2 above)
Comment (Akhill2020) [reply includes/admin/pages.php:276]: on check on demo staging site and it is working as expected.
https://www.screenpresso.com/=o7v6e

**Review by rosinghal — APPROVED** [review]

**Pull request merged**